cmd/common/registry.go

```diff
@@ -259,6 +259,14 @@ func buildCommandFromGadget(
 			// Before running the gadget, we need to get the gadget info to create the
 			// parser based on it
 			if isRunGadget {
+				if len(args) == 0 {
+					if showHelp, _ := cmd.Flags().GetBool("help"); showHelp {
+						additionalMessage := "To see available columns for a specific gadget image, the oci image needs to specified as an argument"
```

Comment:
I think we can make this message more generic. 

```suggestion
						additionalMessage := "Specify the gadget image to get more information about it"
```